src/modulePage/ModulePage.tsx

```diff
@@ -1,12 +1,30 @@
 import React from "react";
+import {RouteComponentProps} from 'react-router';
 import ModuleSection from "./ModuleSection";
 import module8 from '../media/modules/module_8.json';
 import module9 from '../media/modules/module_9.json';
 import module10 from '../media/modules/module_10.json';
 import module11 from '../media/modules/module_11.json';
-const modules = {'computer-vision': module8, 'classification' : module9, 'perceptrons' : module10, 'neural-nets' : module11}
 
-export default function ModulePage(props) {
+export interface ModuleSubsection {
```

Comment:
Strings for the keys aren't necessary in Typescript definitions (or Javascript objects), so you can remove them here if you'd like. (They are necessary for JSON, which can get a bit confusing)